lib/definition.js

```diff
@@ -354,18 +343,18 @@ module.exports = class Definition {
 
   _validateList() {
     assert(this._list_type, `Invalid Env Config (${this._name}): Missing .type()`)
-    const is_array = Array.isArray(this._value)
-    if (this._required && (!is_array || !this._value.length)) {
+    if (!Array.isArray(this._value)) return
```

Comment:
nit: I find coercive checks like `!foo` to be less clear than an explicit boolean check:

```suggestion
    if (Array.isArray(this._value) === false) return
```

Reply:
Yeah, styles are tough.  Personal preference and such.  We typically don't do that style (for no particular reason other than maybe it's more typing?), and I don't have a strong preference on this one.

I could change it here, but it wouldn't be consistent with the other conditionals, and I think for style, that "rule" trumps all--try to follow the style of the overall code.